haversine.py

```diff
@@ -0,0 +1,25 @@
+import math
+
+def haversine(coord1, coord2):
+    meters_to_mile = 1609
```

Comment:
Are these constants? ALL_CAP_WITH_UNDERSCORES for constants

Reply:
```suggestion
    METERS_TO_MILE = 1609
```
Would this be better?

Reply:
Yeah, there was another right below it as well I believe. @Dhadds